packages/application-components/src/components/internals/for-docs/example-wrapper/example-wrapper.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ThemeProvider } from 'emotion-theming';
+import { IntlProvider } from 'react-intl';
+import { customProperties } from '@commercetools-frontend/ui-kit';
+import ModalController from '../modal-controller';
+
+const ExampleWrapper = props => (
+  <ThemeProvider
+    theme={{
+      // Reset theme to default styles, so that the example uses the default uikit tokens
+      fontFamilyDefault: customProperties.fontFamilyDefault,
+      colorSolid: customProperties.colorsSolid,
+    }}
+  >
+    <IntlProvider locale="en">
```

Comment:
So this means I can remove this from the other PR 🙂

Reply:
Yeah, we have it as a central place here since we probably only need it for the rendered examples.
If we also need it for the website, we can then "move it up the tree".